ExpressWash/ExpressWash/WasherController.swift

```diff
@@ -11,6 +11,9 @@ import CoreData
 import CoreLocation
 
 class WasherController {
+
+    static let shared = WasherController()
```

Comment:
I kept wanting to do this too and didn't. But it's probably just easier this way! 👍🏼